build/charts/theia/values.yaml

```
@@ -47,6 +47,33 @@ clickhouse:
     tcpPort: 9000
     # -- HTTP port number for the ClickHouse service.
     httpPort: 8123
+    # As ClickHouse Operator currently does not support the cert management,
+    # we recommend to use Ingress or Load Balancer to expose ClickHouse securely.
+    # But we also provide this option to provide the secure connection from
+    # ClickHouse server end. To use this option, please rotate the certs manually.
+    secureConnection:
+      # -- Determine whether to enable secure connection to ClickHouse
+      enable: false
+      # -- TCP port number for the ClickHouse service.
+      secureTcpPort: 9440
+      # -- HTTP port number for the ClickHouse service.
+      httpsPort: 8443
+      # -- Indicates whether to use auto-generated self-signed TLS certificates.
+      # If false, a Secret named "clickhouse-tls" must be provided with the
+      # following keys: tls.crt and tls.key.
+      selfSignedCert: true
+      # -- Common name to use in the certificate. This is ignored if selfSignedCert
```

I feel it is better to avoid double negatives. Same for below.
```suggestion
      # -- Subject's common name. Only used when selfSignedCert is true.
```